### PR TITLE
Document that cache warmers now must return an array of files to preload

### LINKED ISSUE #13493: [HttpKernel] allow cache warmers to add to the list of preloaded classe…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#36209
| PR author(s) | @nicolas-grekas
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #13493.

### CODE REVIEW

**Review by jmsche — APPROVED** [review]

File: reference/dic_tags.rst [hunk 1]
```diff
@@ -370,6 +370,17 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
         public function warmUp($cacheDirectory)
         {
             // ... do some sort of operations to "warm" your cache
+
+            // this method must return an array with the file paths of the
```
Comment (noniagriconomie) [inline reference/dic_tags.rst:374]: With file paths (without the?)
I am not an english native, but reading the file paths seems weird

File: reference/dic_tags.rst [hunk 2]
```diff
@@ -378,6 +389,11 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
         }
     }
 
+.. deprecated:: 5.1
+
```
Comment (noniagriconomie) [inline reference/dic_tags.rst:393]: Same, is the « the » needed/correct with plural ? Asking a question :)

**Comment by javiereguiluz** [comment]
I've reworded this entirely after asking @nicolas-grekas about this and he explained to me how this works. Thanks!

File: reference/dic_tags.rst (hunk 2 above)
Comment (javiereguiluz) [reply reference/dic_tags.rst:399]: Apparently this is easy ... but in practice it's not. See https://english.stackexchange.com/questions/10854/rule-on-absence-of-the-article-the-with-plural-nouns. I'd keep the current `the` ... but native speakers can remove it if needed in future PRs. Thanks.

**Pull request merged**